modules/core/src/utils/texture.js

```diff
@@ -28,9 +28,19 @@ export function createTexture(layer, image) {
     image = {data: image};
   }
 
+  const specialTextureParameters = {};
```

Comment:
Set default to `null` if image is not compressed

Reply:
`...specialTextureParameter` only works for objects

Reply:
Oh! `...null` is valid syntax! Didn't know that

Reply:
Done